topostats/tracing/disordered_tracing.py

```diff
@@ -297,58 +298,69 @@ def trace_image_disordered(  # pylint: disable=too-many-arguments,too-many-local
     tuple[dict, pd.DataFrame, dict, pd.DataFrame]
         Binary and integer labeled cropped and full-image masks from skeletonising and pruning the grains in the image.
     """
-    # Check both arrays are the same shape - should this be a test instead, why should this ever occur?
-    if image.shape != grains_mask.shape:
-        raise ValueError(f"Image shape ({image.shape}) and Mask shape ({grains_mask.shape}) should match.")
+    # # Check both arrays are the same shape - should this be a test instead, why should this ever occur?
+    # if image.shape != grains_mask.shape:
+    #     raise ValueError(f"Image shape ({image.shape}) and Mask shape ({grains_mask.shape}) should match.")
 
-    cropped_images, cropped_masks, bboxs = prep_arrays(image, grains_mask, pad_width)
-    n_grains = len(cropped_images)
-    img_base = np.zeros_like(image)
+    # cropped_images, cropped_masks, bboxs = prep_arrays(image, grains_mask, pad_width)
```

Comment:
(continued from default config) or need to be padded here otherwise the 3x3 convolutions will not work